lib/ddtrace/contrib/active_support/notifications/subscriber.rb

```diff
@@ -21,7 +21,12 @@ def subscriptions
 
             # Returns whether subscriptions have been activated, via #subscribe!
             def subscribed?
-              subscribed == true
+              !subscriptions.empty?
+            end
+
+            def unsubscribe_all
```

Comment:
Although I've considered doing this exact same thing when I first wrote the subscriber, I don't think we should add this to the public API. Reason for not doing so: this is a feature that should only be used in tests, and never by user applications. Creating additional API exposure here increases the chances a user's application becomes dependent on this feature.

Perhaps the reasonable compromise here is to make this method `private` indicating "don't use this, or your application might break."

Reply:
Agreed.